desktop/libs/libsaml/src/libsaml/backend.py

```diff
@@ -120,6 +121,15 @@ def logout(self, request, next_page=None):
       response = saml_logout(request)
       auth_logout(request)
       return response
+    elif conf.LOGOUT_URL.get():
+      auth_logout(request)
+      # This logic was derived from KNOX.
+      prod_url = "console.altus.cloudera.com"
+      redirect_url = "https://sso.cloudera.com/bin/services/support/api/public/logout"
+      if prod_url not in conf.LOGOUT_URL.get():
+        redirect_url = "https://sso.staging-upgrade.aem.cloudera.com/bin/services/support/api/public/logout"
```

Comment:
The URLs should go in the config

Reply:
it may change in the future. This will be improved.